charactersheet/charactersheet/templates/dm/dmscreen.tmpl.html

```diff
@@ -0,0 +1,333 @@
+<div class="row">
+  <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12">
```

Comment:
For consistency, can you add a title to the page and a small description? All the other modules in character sheet and DM tools do this.